.github/workflows/pre-merge-rust.yml

```diff
@@ -38,7 +38,7 @@ jobs:
   pre-merge-rust:
     runs-on: ubuntu-latest
     strategy:
-      matrix: { dir: ['.', 'lib/bindings/python', 'lib/runtime/examples'] }
+      matrix: { dir: ['.', 'lib/bindings/python', 'lib/runtime/examples', 'launch/dynamo-run'] }
```

Comment:
NOTE: There may be a workspace/crate configuration change that would make running cargo test from root `.` also trigger dynamo-run tests, but I don't know the solution for that off the top of my head, so doing this to add the missed coverage for now.